oaipmhTemplates/ariadne/https___vocabs_acdh_oeaw_ac_at_schema_TopCollection.xml

```diff
@@ -0,0 +1,91 @@
+<rdf:RDF
+    xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+    xmlns:owl="http://www.w3.org/2002/07/owl#"
+    xmlns:crm="http://www.cidoc-crm.org/cidoc-crm/"
+    xmlns:crmpe="http://parthenos.d4science.org/CRMext/CRMpe.rdfs/"
+    xmlns:aocat="https://www.ariadne-infrastructure.eu/resource/ao/cat/1.1/"
+    xmlns:lexvo="http://lexvo.org/id/iso639-2/"
+    xmlns:skos="http://www.w3.org/2004/02/skos/core#"
+    xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
+    xmlns:ariadneplus="https://ariadne-infrastructure.eu/aocat/">
+
+<aocat:AO_Collection val="URI" asAttribute="rdf:about">
+    <aocat:has_identifier val="/acdh:hasNonLinkedIdentifier" count="*"></aocat:has_identifier>
+    <aocat:has_original_id val="/acdh:hasIdentifier" count="*"></aocat:has_original_id>
+    <aocat:has_type></aocat:has_type> <!-- TODO: !-->
+    <aocat:has_title val="/acdh:hasTitle" count="*" lang="true"></aocat:has_title>
+    <aocat:has_description val="/acdh:hasDescription" count="*" lang="true"></aocat:has_description>
+    <aocat:was_issued val="/acdh:hasAvailableDate" count="1" dateFormat="Date"></aocat:was_issued>
+    <aocat:was_created_on val="/acdh:hasAvailableDate" count="1" dateFormat="Date"></aocat:was_created_on> <!-- how is this different from was_issued?-->
+    <aocat:was_modified val="/acdh:hasUpdatedDate" count="*" dateFormat="Date"></aocat:was_modified>
+
+    <aocat:has_language>
+        <aocat:AO_Concept val="acdh:hasLanguage" asAttribute="rdf:about">
+            <skos:prefLabel val="@acdh:hasLanguage/acdh:hasTitle" count="*" lang="true"></skos:prefLabel>
+        </aocat:AO_Concept>
+    <!-- longer TODO: link our lang vocab to lexvo.org -->
+    <!-- and also find a way to inlude the shortcodes instead of the full language-->
+    </aocat:has_language>
+    
+    <aocat:has_landing_page val="acdh:hasPid" asAttribute="rdf:resource"></aocat:has_landing_page>
```

Comment:
Unfortunately, which looks like an issue with the underlying xml libraries, I'm unable to resolve the namespace prefix used inside the `asAttribute` attribute value until the namespace definition isn't repeated in the node :-(
It means for the `asAttribute="rdf:resource"` I need also `xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"` declaration in this node. Alternatively it can be just `asAttribute="http://www.w3.org/1999/02/22-rdf-syntax-ns#resource"`